### DISCUSSION TITLE
Please highlight the active tab more clearly

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#8832] [General]
When connected to multiple peers, highlight the active tab more clearly than now is so as to recognize it more easily
Thank you
![2024-07-25-5678](https://github.com/user-attachments/assets/fed2a1cb-ca43-484f-829c-0a9634b761d2)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/8842
